src/sqlancer/postgres/ast/PostgresBinaryArithmeticOperation.java

```diff
@@ -58,15 +58,21 @@ public PostgresConstant apply(PostgresConstant left, PostgresConstant right) {
         private String textRepresentation;
 
         private static PostgresConstant applyBitOperation(PostgresConstant left, PostgresConstant right,
-                BinaryOperator<Long> op) {
+                BinaryOperator<Float> op) {
```

Comment:
Using `BinaryOperator<Float>` in both an integer and floating-point context seems error-prone. Not sure if we should add another class instead?

Reply:
Sure. I will handle this by adding common classes for these data types in a separate PR.